### DISCUSSION TITLE
Trouble in assembling algae mitogenome

### DISCUSSION THREAD

**Discussion by Alex-132** [Kinggerm/GetOrganelle#194] [Q&A]
I have a question regarding the assembly based mitogenome assembly option on get organelle.

I do not have the best dataset, but it is all that is available for this species at the moment and I am trying to get at least parts of the mitogenome.

I ran the following command:
get_organelle_from_assembly.py -F embplant_mt -g spades/assembly_graph.fastg -o /home/getorganell_1 -t 16 --min-depth 10

I get errors that I couldn't fix yet. Can you help me with that?
I appended the log files.

I would appreciate your help and time.
Kind regards,
Alexandra Schmidt

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9546734/get_org.log.txt)
[slimmed_assembly_graph.slim.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9546735/slimmed_assembly_graph.slim.log.txt)

**Comment by Kinggerm** [comment]
It looks like there is a limited presence of mt in the graph. I would try lowering the min depth first.
